tuf/ngclient/_internal/trusted_metadata_set.py

```diff
@@ -408,7 +360,7 @@ def update_delegated_targets(
         if self.snapshot is None:
             raise RuntimeError("Cannot load targets before snapshot")
 
-        delegator = self.get(delegator_name)
+        delegator: Optional[Metadata] = self.get(delegator_name)
```

Comment:
Are we annotating this as Optional so that mypy can understand that `None` is a valid value for delegator? Why wasn't this required before?

Reply:
No, mypy was not complaining about this. I did it to help the code editor figure it out. I can separate it in another commit or explain it in the commit message if it looks weird in between the other changes.

Reply:
It does look a bit out of place in this commit, though it's not an objectionable change by itself.